introduction.md

```diff
@@ -2,12 +2,8 @@
 headline: <h1>Ethereum is about to become the most <span class="accent">powerful</span>, most <span class="accent">used</span>, most <span class="accent">credibly-neutral</span>, and most <span class="accent accent-green">energy-efficient</span> blockchain network in existence.</h1>
 ---
 
-**Seeking write-up from community. 3-paragraphs or less summarizing the beacon chain and the impending switch to Proof of Stake.**
+The Ethereum network will undergo its arguably most massive upgrade since its inception in 2015, swapping the energy-intensive Proof-of-work (PoW) consensus mechanism to the Proof-of-stake (PoS) one, placing a renewed emphasis on decentralization and security, lowering the hardware cost to run a node as well as laying the groundwork to tackle the scalability issue aka the [blockchain trilemma](https://ethereum.org/en/eth2/vision/)
```

Comment:
I like this introductory sentence - It would be great to have sources for: 
- energy-intensive
- sources for each item in: decentralization, security, and lowering the hardware cost 
- Let's also work "energy efficiency" into the ETH pros as a direct counterpoint to the previous sentence about BTC.